docs/photoniq/fingerprinting/fingerprinting-metrics.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 40
+title: Fingerprinting Metrics
+---
+
+The tracking metrics below quantify the scale of Digital Fingerprinting and can help identify usage trends.
+
+## Stability Rate
```

Comment:
We don't do _Stability Rate_ right now.  But we do _Accuracy Rate_

Use the URL below as a background for Accuracy Rate, but it is to technical.
https://dev.fingerprint.com/docs/understanding-our-995-accuracy

My attempt to rewrite this:

_The percent that the visitor has been correctly identified._  

I am having difficulty coming up with the right words.  See if the above link can help get us what we need for accuracy without having to state formulas here.